tests/console/add_update_test_repo.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add maintenance repositories after installation, is much faster than during installation
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use strict;
+use base "y2logsstep";
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
```

Comment:
I think testapi `get_var_array` does the some

Reply:
thank you, changed